pkg/ingester/stream_rate_calculator.go

```diff
@@ -35,7 +35,7 @@ type StreamRateCalculator struct {
 
 func NewStreamRateCalculator() *StreamRateCalculator {
 	calc := &StreamRateCalculator{
-		size:     defaultStripeSize,
+		size: defaultStripeSize,
```

Comment:
Not sure why these files are included in the PR, I assume because of some auto-formatting run over the whole project. I'd suggest reverting since they're not related to the PR, but if it's a PITA then don't worry.

Reply:
Probably because of the `make lint` I ran. I'll see about reverting it.
Also, do you think we can get this PR on a 2.7.x release? This behavior is wrong and should be backported IMO

Reply:
It was actually a rebase issue. I hope you don't mind me rebasing to fix this, as you already approved the PR

Reply:
Sounds good, I agree it should be backported. No prob about rebasing once a review is complete :+1: thanks!